lib/index.js

```diff
@@ -67,9 +68,24 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
     optionallyAuthenticated,
     // FIXME add an implementation in bedrock-express
     boolParser(),
-    validate({query: 'account.getAccountsQuery'}),
+    intParser(),
+    validate({query: 'account.get'}),
     asyncHandler(async (req, res) => {
-      if(req.query.exists === true && req.query.email) {
+      /**
+       * @func getAccounts
+       * @param {Request} req
+       * @param {Response} res
+       * @param {Object} req.query
+       * @param {String} req.query.email - required to acces this route
+       * @param {Boolean} req.query.exists - used by Registration
+       * @param {Number} req.query.limit
+       * @param  {String} req.query.cursor - an account DID
+       * @description multi-purpose function that can used
+       * by visitors not logged in to confirm an email exists
+       * in the system or by admins to access sensitive account
+       * information
```

Comment:
Recommend something more like this:

```
Gets account records (or just checks for their existence). Access to accounts will be based on the capabilities presented in the request.
```

Reply:
I like this 
```
Gets account records (or checks for their existence).
```
 but the other part is a bit to abstract. I went the docs to be there so someone totally new will get it immediately.